content.js

```diff
@@ -43,10 +34,10 @@ function copyPrDescription() {
     return;
   };
 
-  const commitTitle = `${prTitleEl.value} (#${prNumberMatch.groups['pr_number']})`;
+  const commitTitle = `${prMetadata.title} (#${prNumber})`;
 
   // Remove leading HTML comments
-  let commitBody = prBodyEl.textContent.replace(/^<!--.*?-->\n*/gs, '');
+  let commitBody = prMetadata.body.replace(/^<!--.*?-->\n*/gs, '');
```

Comment:
`body` can be null here if the PR description is empty. I think that wasn't true of the textContent before. Something like this should fix

```js
const prTitle = prMetadata.title || '';
const prBody = prMetadata.body || '';
```